frontend/src/scenes/persons/PersonCohorts.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from 'react'
+import { useActions, useValues } from 'kea'
+import { personsLogic } from './personsLogic'
+import Skeleton from 'antd/lib/skeleton'
+import { Table } from 'antd'
+import { CohortType } from '~/types'
+
+export function PersonCohorts(): JSX.Element {
+    const { cohorts, cohortsLoading } = useValues(personsLogic)
+    const { loadCohorts, navigateToCohort } = useActions(personsLogic)
+
+    useEffect(() => {
+        if (cohorts === null && !cohortsLoading) {
+            loadCohorts()
```

Comment:
What's the reason for having this effect here rather than calling loadCohorts on mount in personslogic?

Reply:
The idea was to be lazy about loading data - this way it only loads when you switch tabs.